.github/workflows/ci.yml

```diff
@@ -21,54 +21,43 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
+          - 'lts'
```

Comment:
Should either keep 1.6 or bump julia compat in Project.toml I guess.

Reply:
Good point, I added min